aslam_offline_calibration/kalibr/python/kalibr_msf_config

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+import kalibr_common as kc
+
+import numpy as np
+import pylab as pl
+import sm
+import argparse
+import sys
+import rospkg
+import yaml
+
+from matplotlib.colors import LogNorm 
+from sm.libsm_python import Transformation
+np.set_printoptions(suppress=True)
+    
+if __name__ == "__main__":        
+    parser = argparse.ArgumentParser(description='Convert camchain files generated for each imu into a msf_config file.')
+    parser.add_argument('--viimu', dest='viYaml', help='Camera configuration as yaml file relative to the vi imu', required=True)
+    parser.add_argument('--mavimu', dest='mavYaml', help='Camera configuration as yaml file relative to the mav imu', required=True)
+    parser.add_argument('--out', dest='outLoc', help='Path to output parameters to', required=False)
+    parsed = parser.parse_args()
+            
+    #load the transforms
+    camchain_SB = kc.ConfigReader.CameraChainParameters(parsed.viYaml)
+    T_SB = camchain_SB.getExtrinsicsImuToCam(0)
+    camchain_B = kc.ConfigReader.CameraChainParameters(parsed.mavYaml)
+    T_B = camchain_B.getExtrinsicsImuToCam(0)
```

Comment:
I would expect a frame pair as subscript here. Personally, I like Paul's view on that: http://paulfurgale.info/news/2014/6/9/representing-robot-pose-the-good-the-bad-and-the-ugly